packages/types/src/Metadata/util/testUtil.ts

```diff
@@ -3,21 +3,14 @@
 // of the Apache-2.0 license. See the LICENSE file for details.
 
 import extrinsicsFromMeta from '@polkadot/api-metadata/extrinsics/fromMetadata';
-import { getTypeRegistry } from '@polkadot/types';
-import * as srmlTypes from '@polkadot/types/srml/definitions';
+import { injectDefinitions } from '@polkadot/types/srml';
```

Comment:
Move this to `srml/index.ts` - it is very clear which definitions we are injecting